### ISSUE TITLE
Wrong typescript declarations for bitable.appTableRecord.list

### ISSUE THREAD

**Issue by felix98765** [larksuite/node-sdk#21]
The API `bitable.appTableRecord.list` (and listWithIterator) returns a plain object in `data.items.fields`, but the typescript declaration said it is a `Map<string, ...>`, which is incorrect, it should be a `Record<...>`.

```ts
fields: Map<string, string | boolean | {
    text?: string;
    link?: string;
} | {
    location?: string;
    pname?: string;
    cityname?: string;
    adname?: string;
    address?: string;
    name?: string;
    full_address?: string;
} | Array<string> | Array<{
    id?: string;
    name?: string;
    en_name?: string;
    email?: string;
}> | Array<{
    file_token?: string;
    name?: string;
    type?: string;
    size?: number;
    url?: string;
    tmp_url?: string;
}>>
```

**Comment by mazhe-nerd** [comment]
I checked the generation logic, it should be record, but I still need to confirm it。you can use` as` to manually declare the type first, I will fix it as soon as possible after confirmation.

**Comment by mazhe-nerd** [comment]
Fixed in 1.8.1
